src/foam/core/Mixin.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Mixin',
+
+  documentation: `
+    Axiom which installs properties from another model. These installed
+    properties behave as if they were defined on the model itself.
+  `,
+
+  properties: [
+    {
+      name: 'name',
+      getter: function() { return 'mixin_' + this.path; }
+    },
+    'flags',
+    'path',
```

Comment:
What is this used for?

Reply:
which one - path? That's the string that's passed into the AxiomArray